### PR TITLE
Runtime 7.4.2 release notes

### PR DESCRIPTION
**Pull request by jwitz**
Not ready yet, waiting on changes from Airflow 2.5.3

### CODE REVIEW

File: astro/runtime-release-notes.md [hunk 1]
```diff
@@ -24,6 +24,24 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
+
+- Release date: 
+- Airflow version: 2.5.3
+
+### Airflow 2.5.3
+
+Astro Runtime 7.2.0 includes same-day support for Apache Airflow 2.5.3. Airflow 2.5.3 contains a number of bug fixes including:
+
+- Fix validation of date-time field in API and Parameter schemas ([29395](https://github.com/apache/airflow/pull/29395))
+- DAG list sorting lost when switching page ([29756](https://github.com/apache/airflow/pull/29756))
```
Comment (jwitz) [inline astro/runtime-release-notes.md:37]:
```suggestion
- Fix `TriggerRuleDep` when the mapped tasks count is 0 ([30084](https://github.com/apache/airflow/pull/30084))
- Fix some long known Graph View UI problems ([29971](https://github.com/apache/airflow/pull/29971))
```

**Review by jedcunningham — APPROVED** [review]

File: astro/runtime-release-notes.md [hunk 2]
```diff
@@ -24,6 +24,24 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
+
+- Release date: 
+- Airflow version: 2.5.3
+
+### Airflow 2.5.3
+
+Astro Runtime 7.2.0 includes same-day support for Apache Airflow 2.5.3. Airflow 2.5.3 contains a number of bug fixes including:
+
+- Fix `TriggerRuleDep` when the mapped tasks count is 0 ([30084](https://github.com/apache/airflow/pull/30084))
+- Fix some long known Graph View UI problems ([29971](https://github.com/apache/airflow/pull/29971))
```
Comment (lzdanski) [inline astro/runtime-release-notes.md:37]:
```suggestion
- Fix some known Graph View UI problems ([29971](https://github.com/apache/airflow/pull/29971))
```

Comment (lzdanski) [reply astro/runtime-release-notes.md:37]: I saw that this was the name of the issue, but probably should go with this phrasing.

File: src/versions.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 export const siteVariables = {
   // version-specific
   cliVersion: '1.13.0',
```
Comment (lzdanski) [inline src/versions.js:3]: There was a merge conflict in this file. Can you confirm that this is the correct CLI version + Runtime version?

**Review by lzdanski — CHANGES_REQUESTED** [review]

File: astro/runtime-release-notes.md [hunk 4]
```diff
@@ -24,6 +24,24 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
 
 To upgrade Astro Runtime, see [Upgrade Astro Runtime](upgrade-runtime.md). For general product release notes, see [Astro Release Notes](release-notes.md). If you have any questions or a bug to report, contact [Astronomer support](https://cloud.astronomer.io/support).
 
+## Astro Runtime 7.4.2
+
+- Release date: April 3, 2023
```
Comment (ephraimbuddy) [inline astro/runtime-release-notes.md:29]:
```suggestion
- Release date: April 3, 2023
```
We released it on Saturday and I think some customers might have seen it

**Review by ephraimbuddy — APPROVED** [review]

**Review by ephraimbuddy — APPROVED** [review]

**Pull request merged**